### PR TITLE
modules: use provided logger

### PR DESCRIPTION
**Pull request by marbar3778**
## Description

Use provided logger instead of creating new ones

### CODE REVIEW

**Review by iverc — APPROVED** [review]

File: x/ethbridge/handler.go [hunk 1]
```diff
@@ -56,28 +39,30 @@ func NewHandler(
...
-			errorMessageKey, err.Error())
+		logger.Error("bridge keeper failed to process claim.",
+			"error:", err.Error())
 		return nil, err
 	}
 	if status.Text == oracle.SuccessStatusText {
-		if err = bridgeKeeper.ProcessSuccessfulClaim(ctx, status.FinalClaim, sugaredLogger); err != nil {
-			sugaredLogger.Errorw("bridge keeper failed to process successful claim.",
-				errorMessageKey, err.Error())
+		if err = bridgeKeeper.ProcessSuccessfulClaim(ctx, status.FinalClaim); err != nil {
```
Comment (ElliotFriedman) [inline x/ethbridge/handler.go:57]: Does this logger format things in a JSON format? That was one of the motivations behind using the sugared logger.

**Review by ElliotFriedman — CHANGES_REQUESTED** [review]

File: x/ethbridge/handler.go (hunk 1 above)
Comment (marbar3778) [reply x/ethbridge/handler.go:57]: you can define any logger you want. It is defined here https://github.com/Sifchain/sifnode/blob/develop/app/app.go#L129. You can have plain text or json by default or define your own logger by implementing the interface.

Comment (marbar3778) [reply x/ethbridge/handler.go:57]: Here are some docs I recommend reading https://docs.tendermint.com/master/nodes/logging.html and here you can see the change you can make to define json.

Comment (ElliotFriedman) [reply x/ethbridge/handler.go:57]: I read that doc and it did not mention JSON anywhere in it.

To use JSON, do I need to build a JSON object, stringify it, then log it out?

Comment (marbar3778) [reply x/ethbridge/handler.go:57]: The one that would be used on the current chain is https://github.com/go-kit/kit/blob/master/log/json_logger.go. You can change a  node that is currently running to json format. You may need to define string methods if they are not defined, but if zap was working so will this.  The new version of the sdk uses zerolog.

File: x/ethbridge/handler.go [hunk 2]
```diff
@@ -56,28 +39,30 @@ func NewHandler(
...
+			logger.Error("bridge keeper failed to process successful claim.",
+				"error:", err.Error())
```
Comment (banshee) [inline x/ethbridge/handler.go:59]: Why did you switch this to using a string instead of errorMessageKey?

Comment (marbar3778) [reply x/ethbridge/handler.go:59]: I can revert. I found it confusing this was a key, but makes sense for your logging system.

Comment (marbar3778) [reply x/ethbridge/handler.go]: reverted

File: x/clp/handler.go [hunk 3]
```diff
@@ -2,12 +2,13 @@ package clp
 
 import (
 	"fmt"
```
[the remark this answers is not in the document]
Comment (marbar3778) [reply x/clp/handler.go:4]: its still used here 

```
			errMsg := fmt.Sprintf("unrecognized %s message type: %T", ModuleName, msg)
			return nil, sdkerrors.Wrap(sdkerrors.ErrUnknownRequest, errMsg)
```

**Review by iverc — APPROVED** [review]

**Review by iverc — APPROVED** [review]

**Pull request closed without merging**